services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs.py

```diff
@@ -78,9 +81,10 @@ async def list_jobs(
     )
     _logger.debug("Listing Jobs in Solver '%s'", solver.name)
 
-    projects: list[Project] = await webserver_api.list_projects(solver.name)
+    projects_page = await webserver_api.list_projects(solver.name, limit=20, offset=0)
```

Comment:
obvious question: why did you create an additional entrypoint vs extending this one to allow for paging?
how is the paging done? not via query parameters? ah ok. sorry just read better, this is because of the response model I see.

Reply:
- the current version of `list_jobs` was designed to return a `list[Jobs]` which is incompatible iwth pagination (that returns an object that includes that list). The only way to preserve backwards compatibliity here is to create a separate entry. 
- we will wrap this in the python client and mark then the list as deprecated
- we use offset-limit strategy of pagination. Uses query paramenter and return an object with items and links (see fastapi_pagination doc for more details(